packages/lodestar-cli/src/cmds/beacon/config.ts

```diff
@@ -32,11 +32,7 @@ export async function writeBeaconConfig(filename: string, config: Partial<IBeaco
  * This needs to be a synchronous function because it will be run as part of the yargs 'build' step
  */
 export function readBeaconConfig(filename: string): Partial<IBeaconNodeOptions> {
-  try {
-    return readFileSync(filename) as Partial<IBeaconNodeOptions>;
-  } catch(e) {
-    return {};
-  }
+  return readFileSync(filename) as Partial<IBeaconNodeOptions>;
```

Comment:
I think this diff is the only one needed for this PR. The others should be removed

Reply:
done.

Reply:
done.